package.json

```diff
@@ -8,26 +8,28 @@
   "description": "What am I about?",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "node server.js",
+    "dev": "nodemon server.js"
   },
   "dependencies": {
```

Comment:
use carets for updating deps rather than strict versioning.  that way the repo doesn't need to be updated every few months unless there are major version changes

Reply:
Just to clarify, are you saying I should write `"cross-env": "^7.0.0"` instead of what it is now etc...?